src/main/java/seedu/duke/account/MoneyCommand.java

```diff
@@ -0,0 +1,134 @@
+package seedu.duke.account;
+
+import java.util.List;
+
+import seedu.duke.CurrencyList;
+import seedu.duke.CurrencyStructure;
+import seedu.duke.Wallet;
+import seedu.duke.WalletFile;
+import seedu.duke.exception.FinanceException;
+import seedu.duke.exception.FinanceException.ExceptionCollection;
+
+public class MoneyCommand {
```

Comment:
It's more preferred if each sub-command is a class (Single Responsibly Principle) that inherits from a more general class.